Resources/Prototypes/Nyanotrasen/Entities/Mobs/NPCs/animals.yml

```diff
@@ -8,4 +8,112 @@
     sprite: Nyanotrasen/Mobs/Pets/whiteferret.rsi
     layers:
     - map: ["enum.DamageStateVisualLayers.Base"]
-      state: ferret
\ No newline at end of file
+      state: ferret
+
+- type: entity
+  name: mothroach
+  parent: SimpleMobBase
+  id: MobMothroach
+  description: This is the adorable by-product of multiple attempts at genetically mixing mothpeople with cockroaches.
+  components:
+  - type: GhostTakeoverAvailable
+    makeSentient: true
+    allowSpeech: true
+    allowMovement: true
+    whitelistRequired: false
+    name: ghost-role-information-mothroach-name
+    description: ghost-role-information-mothroach-description
+  - type: Speech
+    speechSounds: Chitter
+  - type: Sprite
+    drawdepth: Mobs
+    sprite: Nyanotrasen/Mobs/Animals/mothroach.rsi
+    layers:
+    - map: ["enum.DamageStateVisualLayers.Base"]
+      state: mothroach
+  - type: Item
+    size: 10
+  - type: Clothing
+    quickEquip: false
+    sprite: Nyanotrasen/Mobs/Animals/mothroach.rsi
+    equippedPrefix: 0
+    slots:
+    - HEAD
+  - type: Physics
+  - type: Fixtures
+    fixtures:
+    - shape:
+        !type:PhysShapeCircle
+        radius: 0.3
+      density: 70
+      mask:
+      - MobMask
+      layer:
+      - MobLayer
+  - type: MobState
+  - type: MobThresholds
+    thresholds:
+      0: Alive
+      25: Critical
+      50: Dead
+  - type: MovementSpeedModifier
+    baseWalkSpeed : 5
```

Comment:
no reason not to allow a lower walk speed, since only players use it

Reply:
Same values as the mouse